### PR TITLE
Fixes #5379: Remove footer field from theme submission form

### LINKED ISSUE #5379: Remove footer field from theme submission form
### Describe the problem and steps to reproduce it:
Theme footers are obsolete, so we should remove the field on the theme submission form where we ask for a footer image.

### Anything else we should know?
https://www.dropbox.com/s/k5gvodd0c88pazg/Screenshot%202017-05-10%2011.31.23.png?dl=0

### PR DESCRIPTION
**Pull request by CuriousLearner**
Fixes #5379 

Removed footer field from Theme submission form

### CODE REVIEW

**Comment by CuriousLearner** [comment]
Fixes #5379 

@eviljeff Can you please have a look at this?

**Comment by CuriousLearner** [comment]
Hey @EnTeQuAk !

Any updates here? :)

**Comment by EnTeQuAk** [comment]
Sorry, I missed the notifications, I'll have a look shortly

File: src/olympia/addons/tasks.py [hunk 1]
```diff
@@ -286,28 +285,23 @@ def theme_checksum(theme, **kw):
...
 @task
 @write
-def save_theme(header, footer, addon, **kw):
+def save_theme(header, addon, **kw):
     """Save theme image and calculates checksum after theme save."""
     dst_root = os.path.join(user_media_path('addons'), str(addon.id))
     header = os.path.join(settings.TMP_PATH, 'persona_header', header)
     header_dst = os.path.join(dst_root, 'header.png')
-    if footer:
-        footer = os.path.join(settings.TMP_PATH, 'persona_footer', footer)
```
Comment (EnTeQuAk) [inline src/olympia/addons/tasks.py]: If this lands we should remove all files in that folder too (nothing you can do though)

File: src/olympia/devhub/tests/test_forms.py [hunk 2]
```diff
@@ -672,11 +648,12 @@ def test_reupload_legacy_header_only(self, make_checksum_mock):
         - On approving, it would see 'footer.png' !== 'leg.png'
         - It run move_stored_file('footer.png', 'leg.png').
         - But footer.png does not exist. BAM BUG.
+
+        Footer has been removed in issue # 5379
```
Comment (EnTeQuAk) [inline src/olympia/devhub/tests/test_forms.py]: Let's add a link to the issue if possible.

File: src/olympia/reviewers/models.py [hunk 3]
```diff
@@ -1148,7 +1148,6 @@ def get_queryset(self):
 class RereviewQueueTheme(ModelBase):
     theme = models.ForeignKey(Persona)
     header = models.CharField(max_length=72, blank=True, default='')
-    footer = models.CharField(max_length=72, blank=True, default='')
```
Comment (EnTeQuAk) [inline src/olympia/reviewers/models.py]: This needs a migration in `src/olympia/migrations/` that removes this field from the database. This should not be done in this PR though but as a follow-up. Can you file an issue about that?

**Comment by CuriousLearner** [comment]
@EnTeQuAk I'm done with that nitpick. Also, I've created this issue: https://github.com/mozilla/addons-server/issues/7858

File: settings.py [hunk 4]
```diff
@@ -80,7 +80,7 @@
 SERVICES_DOMAIN = urlparse(SITE_URL).netloc
 SERVICES_URL = SITE_URL
 
-ALLOWED_HOSTS = ALLOWED_HOSTS + [SERVICES_DOMAIN]
+ALLOWED_HOSTS = ALLOWED_HOSTS + [SERVICES_DOMAIN] + ['localhost']
```
Comment (EnTeQuAk) [inline settings.py]: I think you submitted that accidentally, mind backing that change out?

File: docs/settings/local_settings.dev.py [hunk 5]
```diff
@@ -1,3 +1,3 @@
 from settings import *  # noqa
 
-INTERNAL_IPS = ('127.0.0.1',)
+INTERNAL_IPS = ('127.0.0.1', 'localhost')
```
Comment (EnTeQuAk) [inline docs/settings/local_settings.dev.py]: I think you submitted that accidentally, mind backing that change out?

File: src/olympia/devhub/tests/test_forms.py [hunk 6]
```diff
@@ -672,11 +648,13 @@ def test_reupload_legacy_header_only(self, make_checksum_mock):
         - On approving, it would see 'footer.png' !== 'leg.png'
         - It run move_stored_file('footer.png', 'leg.png').
         - But footer.png does not exist. BAM BUG.
+
+        Footer has been removed in issue Issue #5379
```
Comment (EnTeQuAk) [inline src/olympia/devhub/tests/test_forms.py]: duplicate "Issue"

**Comment by CuriousLearner** [comment]
@EnTeQuAk Oops! Sorry I accidentally added it. Fixed all those things.

**Pull request merged**